src/components/Votes/Votes.test.tsx

```diff
@@ -62,4 +126,40 @@ describe("Votes", () => {
       expect(storeDispatchSpy).toHaveBeenCalledWith(ActionFactory.deleteVote("test-id"));
     });
   });
+
+  describe("should dispatch to store on button press", () => {
+    const storeDispatchSpy = jest.spyOn(store, "dispatch");
+
+    test("addVote", () => {
+      const votes = render(createVotes(true, true));
+      fireEvent.click(votes.container.getElementsByClassName("dot-button")[1]);
+      expect(storeDispatchSpy).toHaveBeenCalledWith(ActionFactory.addVote("test-id"));
+    });
+
+    test("deleteVote", () => {
+      const votes = render(createVotes(true, true));
+      fireEvent.click(votes.container.getElementsByClassName("dot-button")[0]);
+      expect(storeDispatchSpy).toHaveBeenCalledWith(ActionFactory.deleteVote("test-id"));
+    });
+  });
+
+  describe("Test amount of visible votes", () => {
+    test("test-user-1 has one vote during vote phase", () => {
+      Parse.User.current = jest.fn(() => ({id: "test-user-1"}));
+      const {container} = render(createNote());
+      expect((container.querySelector(".dot-button")?.lastChild as HTMLSpanElement).innerHTML).toEqual("1");
+    });
+
+    test("test-user-2 hast two votes during vote phase", () => {
+      Parse.User.current = jest.fn(() => ({id: "test-user-2"}));
+      const {container} = render(createNote());
+      expect((container.querySelector(".dot-button")?.lastChild as HTMLSpanElement).innerHTML).toEqual("2");
+    });
+
+    test("test-user-3 hast zero votes during vote phase and has only the add vote button", () => {
+      Parse.User.current = jest.fn(() => ({id: "test-user-3"}));
+      const {container} = render(createNote());
+      expect(container.querySelector(".note__votes")?.childElementCount).toEqual(1);
+    });
+  });
```

Comment:
This should probably be tested in the Note test suite because the Note is reponsible for filtering the votes

Reply:
Correct. Changed this.